mm2src/coins_activation/src/l2/init_l2_error.rs

```diff
@@ -62,6 +62,9 @@ impl From<CoinConfWithProtocolError> for InitL2Error {
             CoinConfWithProtocolError::UnexpectedProtocol { ticker, protocol } => {
                 InitL2Error::UnexpectedL2Protocol { ticker, protocol }
             },
+            CoinConfWithProtocolError::ProtocolMissMatch { ticker, .. } => {
+                InitL2Error::Internal(format!("Protocol from request is not supported for {}", ticker))
+            },
```

Comment:
nit: We might want to print out the two protocols here.

Reply:
This means that setting the protocol from a request is not supported for L2 as there is no custom tokens for L2, and we set it as none below. So no need to print out the protocol IMHO.
https://github.com/KomodoPlatform/komodo-defi-framework/blob/c7638a578cee3063c30df201039fb6725fe135df/mm2src/coins_activation/src/l2/init_l2.rs#L82

Reply:
no idea what i event meant here xD, resolving